app/assets/javascripts/modules/soloteam.js.coffee

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2008-2016, Puzzle ITC GmbH. This file is part of
+# Cryptopus and licensed under the Affero General Public License version 3 or later.
+# See the COPYING file at the top-level directory or at
+# https://github.com/puzzle/cryptopus.
+
+app = window.App ||= {}
```

Comment:
Warum verwendet diese Klasse statische Methoden? Wir sollten uns auf eine Konvention einigen, entweder wie in den vorhergehenden Files (dafür würd' ich stimmen) oder wie hier.